.ci.yaml

```diff
@@ -45,9 +45,6 @@ platform_properties:
       device_type: none
       os: Windows-10
 
-# The current android emulator config names can be found here:
-# https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/android/avd/proto
-# You may use those names for the android_virtual_device version.
 targets:
```

Comment:
Removed this comment because `android_virtual_device` is not even referenced here and updated documentation about upgrading this dependency will land with https://github.com/flutter/flutter/pull/151601.